lib/Data/Database.php

```diff
@@ -865,4 +973,25 @@ private static function _upgradeDatabase($oldversion)
                 );
         }
     }
+
+    /**
+     * read CLOB for OCI
+     * https://stackoverflow.com/questions/36200534/pdo-oci-into-a-clob-field
```

Comment:
I'm not sure what the correct PHPDoc syntax for links is, but it should be sth. like this:

```suggestion
     * { @link https://stackoverflow.com/questions/36200534/pdo-oci-into-a-clob-field }
```